README.md

```diff
@@ -25,3 +25,112 @@ Swagger configuration params that can be set in microfocus-config.js:
  - deepLinking: true
 
 For a complete list of swagger configuration params refer [Swagger configuration params](https://github.com/swagger-api/swagger-ui/blob/master/docs/usage/configuration.md#core).
+
+## Using this module in Spring Boot
+
+[Here](https://github.com/CAFapi/caf-swagger) is an example of how this swagger-ui can be used in a Spring Boot application : 
```

Comment:
Move the link in a level to `https://github.com/CAFapi/caf-swagger/tree/master/caf-swagger-ui`.